### ISSUE TITLE
Accessibility issues with date/time pickers in admin

### ISSUE THREAD

**Issue by dominicduffin1** [forem/forem#11999]
**Is your feature request related to a problem? Please describe.**

The `select` elements generated by `f.datetime_select` in, for example, `/admin/events/new` and `/admin/sponsorships/:id/edit` fail the axe accessibility test for "Select element must have an accessible name".

![2020-12-21 17 13 26 localhost 9f22711a3c40](https://user-images.githubusercontent.com/26224873/102803940-f5148580-43b0-11eb-8b3b-0b5e7a719f11.png)
![2020-12-21 17 14 32 localhost bf6a0f9dc076](https://user-images.githubusercontent.com/26224873/102803953-f940a300-43b0-11eb-9f9b-8c70abdfa1b3.png)

**Describe the solution you'd like**

The solution needs to fix at least one of the issues from axe:

- aria-label attribute does not exist or is empty

- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

- Form element does not have an implicit (wrapped) <label>

- Form element does not have an explicit <label>

- Element has no title attribute

- Element's default semantics were not overridden with role="none" or role="presentation"

I am not sure whether any of these can be accomplished within the functionality of the `datetime_select` method or whether an alternative way of generating the date/time pickers would be necessary to solve this issue.

**Additional context**

This issue came up during work on issue #11392 pr #11979

**Comment by nickytonline** [comment]
After digging into this, it appears to be an issue in Rails itself with `f.datetime_select`. If someone in the community is interested in looking into this, it could be a great first contribution to Rails. Please check their repository first though in case it has already been addressed.

**Comment by mikeydorje** [comment]
Closing as stale. No activity since Feb 2021.

If this issue persists on current DEV or your instance, please reopen with updated details/reproduction steps.
